avionics/common/src/avionics.cpp

```diff
@@ -134,13 +134,13 @@ int main(void) {
 
     switch (init_status) {
     case RocketStatus::NOMINAL:
-        Hal::digitalWrite(Pin::BUILTIN_LED, Hal::PinDigital::LOW);
```

Comment:
Not sure if some of these changes will cause merge conflicts due to #143 being merged; remember to rebase on dev or something before un-"draft"ing this PR